utils/utils.jsx

```diff
@@ -1368,10 +1369,10 @@ export function getUserIdFromChannelName(channel) {
 }
 
 // Used to get the id of the other user from a DM channel id (id1_id2)
-export function getUserIdFromChannelId(channelId) {
+export function getUserIdFromChannelId(channelId, currentUserId = getCurrentUserId(store.getState())) {
```

Comment:
Nit: Do we need this as second argument for this func?

Reply:
Yes, we shouldn't really be accessing the store directly in any utils functions but I didn't want to have to fix every usage of this function to pass in the current user ID, hence the default parameter